detection-rules/attachment_double_encoded_zip_in_html_smuggle.yml

```diff
@@ -39,6 +39,6 @@ source: |
   )
 tags:
   - "Suspicious attachment"
-  - "HTML Smuggling"
+  - "HTML smuggling"

```

Comment:
file name should prob end with "smuggling" vs "smuggle"

Reply:
ty!